modules/login/templates/form_requestaccount.tpl

```diff
@@ -1,65 +1,81 @@
+
```

Comment:
can you take out the blank line and add an `{if $captcha_key}` around the external script loading, similarly to the other html below, so that we don't load things from Google when it's unnecessary?